### PR TITLE
polished dashboard; deleted infinity function in pipeline/extract

### LINKED ISSUE #252: polish dashboard
## Task Description

- move sections into different pages to declutter

## User Stories

- As a user of the dashboard, I want the pages to be clean, simple, so that they are easy to navigate.

## Relevant Files [If Available] 

-

### PR DESCRIPTION
**Pull request by RachelPun**
- polished dashboard (moved sections into different pages to declutter)
- side task: deleted infinity function in pipeline/extract since it's not used
closes #252

### CODE REVIEW

File: dashboard/pages/1_Charts.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                    get_data_from_db,
+                    get_filtered_data,
+                    get_gender_donut_chart,
+                    get_waffle_chart,
+                    get_judge_count_line_chart,
+                    get_verdicts_stacked_bar_chart,
+                    get_case_count_line_chart)
+
+load_dotenv()
+CONN = get_db_connection(ENV)
```
Comment (ErvinRex) [inline dashboard/pages/1_Charts.py:15]: Should we try avoid our connection being a global variable?

File: pipeline/extract.py [hunk 2]
```diff
@@ -36,15 +36,6 @@ def get_stored_titles(conn) -> list:
...
 
-def get_index_to_infinity():
-    """Get index to infinity."""
-
-    index = 1
-    while True:
-        yield index
-        index += 1
-
-
```
Comment (luciegamby-sigmalabs) [inline pipeline/extract.py:47]: Thanks for removing this! Should have caught this earlier

File: dashboard/pages/1_Charts.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    if isinstance(filtered_data, str):
+        st.write(filtered_data)
+    else:
+        st.pyplot(get_waffle_chart(filtered_data))
+
+with row_1[1]:
+    st.subheader("Verdicts By Circuit")
+    st.altair_chart(get_verdicts_stacked_bar_chart(data))
+
+with row_1[2]:
```
Comment (ErvinRex) [inline dashboard/pages/1_Charts.py:57]: great use of rows to clearly define separation of charts.

File: dashboard/streamlit_app.py [hunk 4]
```diff
@@ -255,10 +256,18 @@ def get_data_from_db(conn: connect) -> pd.DataFrame:
 
     get_sidebar()
 
+    show_pages(
```
Comment (ErvinRex) [inline dashboard/streamlit_app.py:259]: Nice addition to easily go through the dashboard!

**Review by ErvinRex — APPROVED** [review]
Overall really well done on getting the dashboard to be more easily understood and walked through. Just make sure we close the connection to the database at the end.

**Pull request merged**